examples/objectctl/pkg/createcmd/create.go

```diff
@@ -0,0 +1,55 @@
+package createcmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/peterbourgon/ff/v4"
+	"github.com/peterbourgon/ff/v4/examples/objectctl/pkg/rootcmd"
+)
+
+type CreateConfig struct {
+	*rootcmd.RootConfig
+	Overwrite bool
+	Flags     *ff.CoreFlags
+	Command   *ff.Command
+}
+
+func New(rootConfig *rootcmd.RootConfig) *CreateConfig {
+	var cfg CreateConfig
+	cfg.RootConfig = rootConfig
+	cfg.Flags = ff.NewFlags("create").SetParent(cfg.RootConfig.Flags)
```

Comment:
I don't want to make SetParent part of the Flags interface, because I don't want to dictate that flag sets need to support the concept of parents. At the moment it's a method on the CoreFlags type, and takes a parent CoreFlags value as a parameter. That means that you need to keep track of the concrete CoreFlags value of parent commands somehow, you can't just rely on the Flags field in the parent command directly. At least, not without type assertions. This is annoying, but I'm not sure what a better approach would be.